dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A database browser for SQLite"
+DESCRIPTION="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite. \
+It is for users and developers wanting to create databases, search, and edit data. \
+It uses a familiar spreadsheet-like interface, and you don't need to learn complicated\ SQL commands.\
+HOMEPAGE="http://sqlitebrowser.org/"
+COPYRIGHT="COPYRIGHT 2012-2014 Martin Kleusberg"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/sqlitebrowser/sqlitebrowser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d0d2e06a69927ba1d0b955f3261ce70c61befc5bd5ddaa06752dae8bb4219ed8"
+SOURCE_FILENAME="sqlitebrowser-$portVersion.tar.gz"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+    sqlitebrowser = $portVersion
+    libs:antlr-2.7.7
+    libs:qcustomplot-source
+    libs:qhexedit
+    libs:qscintilla
```

Comment:
Yes, you should only need to add something like app:sqlitebrowser, but Haikuporter will tell you what is missing. So remove these libs:* lines, and build with -S.